crates/core/tedge_mapper/src/sm_c8y_mapper/tests.rs

```diff
@@ -48,11 +48,7 @@ async fn mapper_publishes_a_supported_operation_and_a_pending_operations_onto_c8
     mqtt_tests::assert_received(
         &mut messages,
         TEST_TIMEOUT_MS,
-        vec![
-            "114,c8y_SoftwareUpdate,c8y_LogfileRequest\n",
-            "118,software-management\n",
-            "500\n",
-        ],
+        vec!["118,software-management\n", "500\n"],
```

Comment:
This is okay for now but this highlights something that needs to be clarified. This `118` message is related to the protocol of the `c8y_LogfileRequest` operation. This make senses because the `c8y_mapper` is no more responsible of the operation declaration but only of the implementation of some of them. But this is surprising.